types/migrations/20250923142543_add-menu.sql

```diff
@@ -0,0 +1,45 @@
+-- Menus
+create table menu (
+    id autouuid primary key,
+    name text not null,
+    slug text not null unique,
+    items jsonb not null
+);
+
+-- Seed default navigation menu (JSONB)
+insert into menu (id, name, slug, items) values (
```

Comment:
if possible, it would be preferable to put _data_ migration code in our rust migration module